interop-reactiveStreams/shared/src/main/scala/scalaz/zio/interop/reactiveStreams.scala

```diff
@@ -6,13 +6,24 @@ import scalaz.zio.stream.{ ZSink, ZStream }
 
 package object reactiveStreams {
```

Comment:
This file should be named `package.scala`, and should be inside the `reactiveStreams` folder.